package.json

```diff
@@ -4,14 +4,14 @@
   "description": "COD Barefoot Nomad Backend Development",
   "main": "index.js",
   "scripts": {
+    "test": "nyc --reporter=lcov --reporter=text-summary mocha --recursive --exit --timeout 10000 --require @babel/register",
     "build": "babel ./src/index.js -d build",
     "start": "npm run build && node build/index.js",
     "dev": "nodemon --exec babel-node src/index.js",
     "createDb": "npx sequelize db:create",
     "dbMigrate": "npx sequelize-cli db:migrate",
     "undoMigrate": "npx sequelize-cli db:migrate:undo",
     "makeSeed": "npx sequelize-cli db:seed:all",
-    "test": "mocha --recursive --exit --timeout 10000 --require @babel/register",
```

Comment:
Would It be better if we still had this script and another separate one for getting coverage

Reply:
@SergeNm please look at the comment above and address asap

Reply:
main test script should always show coverage

Reply:
@didasmbalanya main test script is now always showing coverage